main.go

```diff
@@ -114,6 +119,9 @@ func main() {
     ln := c.QueryParam("ln")
     lnurlpUrl, keysendUrl, nostrUrl, err := ToUrl(ln)
     if err != nil {
+			logger.WithFields(log.Fields{
+				"lightningAddress": ln,
```

Comment:
Please change all camelcase to snake_case, to be consistent with our datadog log format.

Reply:
Done!